README.md

```diff
@@ -17,109 +17,6 @@ OpenSoundscape is currently in active development. If you find a bug, please sub
-* Build the virtual environment for opensoundscape: `poetry install`
-  - If poetry install outputs the following error, make sure to download Python 3.7:
-    ```
-    Installing build dependencies: started
-    Installing build dependencies: finished with status 'done'
-    opensoundscape requires Python '>=3.7,<4.0' but the running Python is 3.6.10
-    ```
-    If you are using `conda`, install Python 3.7 using `conda install python==3.7`
-  - If you are on a Mac and poetry install fails to install `numba`, contact one
-    of the developers for help troubleshooting your issues.
-* Activate the virtual environment with the name provided at install e.g.: `workon opensoundscape-dxMTH98s-py3.7` or `poetry shell`
-* Check that OpenSoundscape runs: `opensoundscape -h`
-* Run tests (from the top-level directory): `poetry run pytest`
-* Go back to your system's Python when you are done: `deactivate`
-
-### Jupyter
-To use OpenSoundscape within JupyterLab, you will have to make an `ipykernel`
-for the OpenSoundscape virtual environment.
-
-- Activate poetry virtual environment, e.g.: `workon opensoundscape-dxMTH98s-py3.7`
-    - Use `poetry env list` if you're not sure what the name of the environment is
-- Create ipython kernel: `python -m ipykernel install --user --name=[name of poetry environment] --display-name=OpenSoundscape`
-- Now when you make a new document on JupyterLab, you should see a Python kernel available called OpenSoundscape.
-- Contributors: if you include Jupyter's `autoreload`, any changes you make to the source code
-  installed via poetry will be reflected whenever you run the `%autoreload` line magic in a cell:
-    ```
-    %load_ext autoreload
-    %autoreload
-    ```
-
-### Contributing to code
-
-Make contributions by editing the code in your fork. Create branches
-for features using `git checkout -b feature_branch_name` and push these
-changes to remote using `git push -u origin feature_branch_name`. To merge a
-feature branch into the development branch, use the GitHub
-web interface to create a merge request.
-
-When contributions in your fork are complete, open a pull request using the
-GitHub web interface. Before opening a PR, do the following to
-ensure the code is consistent with the rest of the package:
-* Run tests: `poetry run pytest`
-* Format the code with `black` style (from the top level of the repo): `poetry run black .`
-  * To automatically handle this, `poetry run pre-commit install`
-* Additional libraries to be installed should be installed with `poetry add`, but
-  in most cases contributors should not add libraries.
-
-### Contributing to documentation
-
-Build the documentation using either poetry or sphinx-build
-- With poetry: `poetry run build_docs`
-- With sphinx-build: `sphinx-build doc doc/_build`
+OpenSoundscape should be run on Python 3.7 and can be installed via pip: `pip opensoundscape==0.4.6`. For more detailed instructions and tutorials, see the documentation.
```

Comment:
maybe "has been tested on Python 3.7" 
I think 3.8 and maybe even 3.9 are probably fine if someone has a reason to use them

Reply:
Good point, made this update